### PR TITLE
New package: dumb-1.0

### PR DESCRIPTION
**Pull request by beefcurtains**
I've successfully built and ran a frontend for this, it produces quality vibrations.

### CODE REVIEW

File: srcpkgs/libdumb/template [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Template file for 'libdumb'
+pkgname=libdumb
+version=1.0
+revision=1
+
+# dumb breaks xlint with its directory structure
+wrksrc=dumb-${version}
```
Comment (xtraeme) [inline srcpkgs/libdumb/template]: This is wrong... please use:

```
wrksrc=dump-${version}
build_wrksrc=cmake
```

**Comment by xtraeme** [comment]
The package is ok, but would be nice if the generated shared library is `versioned`, i.e `libfoo.so.major.minor` rather than `libfoo.so`.

Otherwise xbps won't be able to detect soname bumps.

**Comment by beefcurtains** [comment]
Agreed. Dumb doesn't do that, what do you suggest? ``libdumb.so.1`` and in ``-devel`` a symlink ``libdumb.so -> libdumb.so.1``?

**Comment by xtraeme** [comment]
The build system must be changed to use `-Wl,-soname libdumb.so.1` and then you can make a `libdumb.so` symlink and move it to `-devel`.

**Comment by beefcurtains** [comment]
:heart: :heart: Fixed, tested, works perfectly! Thank you *SO MUCH*! :heart: :heart:

**Comment by beefcurtains** [comment]
Please don't merge this yet, I should change the name back to ``dumb`` first (renamed it before filing this PR because I thought that'd be better but it violates the naming policy). I'd also like to take a stab at including an example frontend (e.g. dumb2wav).

File: srcpkgs/dumb/template [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+esac
+
+pre_install() {
+	vdoc ../readme.txt
+	vdoc ../release.txt
+	vlicense ../licence.txt
+}
+
+dumb-devel_package() {
+	depends="${pkgname}>=${version}_${revision}"
```
Comment (xtraeme) [inline srcpkgs/dumb/template]: rather than `pkgname` you can use `sourcepkg` that expands to the main pkg name.

Comment (pullmoll) [inline srcpkgs/dumb/template]: I also made this type of mistake (`pkgname` instead of `sourcpkg`) in several packages and will identify and update them...

Comment (beefcurtains) [inline srcpkgs/dumb/template]: [A change](https://github.com/chneukirchen/xtools/commit/0f2ee339ae133e7de9045d3c7d480335a8144cd7) has already landed in xtools that should prevent this in the future.

**Comment by beefcurtains** [comment]
There's stuff in there that shouldn't be in there, not sure how that happened.

**Comment by beefcurtains** [comment]
It's not pushing the commits:
```
$ git log|head -n 1
commit e477feeff8b7edd773f93df8e794a4d90d6d1994
$ git push --force origin dumb
Everything up-to-date
```

**Comment by beefcurtains** [comment]
Please re-review :space_invader:

**Pull request merged**